Core/clim-core/commands/tables.lisp

```diff
@@ -221,41 +220,27 @@ designator) inherits menu items."
         (eq inherit-menu :menu))))
 
 (defun %add-menu-item (command-table item after)
-  (with-slots (menu)
-      command-table
+  (with-slots (menu) command-table
     (when (null menu)
       (setf after :start))
     (case after
-      (:start (push item menu))
-      ((:end nil) (setf menu (nconc menu (list item))))
-      (:sort (setf menu (sort (cons item menu)
-                              #'string-lessp
-                              :key #'command-menu-item-name)))
-      (t (push item
-               (cdr (member after menu
-                            :key #'command-menu-item-name
-                            :test #'string-equal))))))
-  (when-let ((keystroke (slot-value item 'keystroke)))
-    (%add-keystroke-item command-table keystroke item nil)))
-
-;; At this point we should still see the gesture name as supplied by the
-;; programmer in 'gesture'
-(defun %add-keystroke-item (command-table gesture item errorp)
-  (with-slots (keystroke-accelerators keystroke-items)
-      command-table
-    (let* ((gesture (if (and (symbolp gesture) ; symbolic gesture name?
-                             (gethash gesture *gesture-names*))
-                        gesture
-                        (multiple-value-list (realize-gesture-spec :keyboard gesture))))
-           (in-table (position gesture keystroke-accelerators :test #'equal)))
-      (when (and in-table errorp)
-        (error 'command-already-present :command-table-name
-               (command-table-designator-as-name command-table)))
-      (if in-table
-          (setf (nth in-table keystroke-items) item)
-          (progn
-            (push gesture keystroke-accelerators)
-            (push item keystroke-items))))))
+      (:start
+       (push item menu))
+      ((:end nil)
+       (setf menu (nconc menu (list item))))
+      (:sort
+       (flet ((menu-name-lessp (x y)
+                (cond ((null x) t)
+                      ((null y) nil)
+                      (t (string-lessp x y)))))
+         (setf menu (sort (cons item menu) #'menu-name-lessp
+                          :key #'command-menu-item-name))))
+      (t
+       (check-type after string)
+       (push item
+             (cdr (member after menu
+                          :key #'command-menu-item-name
+                          :test #'string-equal)))))))
```

Comment:
`(string-equal "NIL" nil) => T` so maybe `menu-name-equal` is needed?

Reply:
I'm not sure why I've put string-equal here, I think that ordinary equal will suffice.

fixed accordingly, but not closing the issue in case if you think it is not a good idea.

Reply:
The old code used `string-equal` but I'm not sure why.

Reply:
Ah, the spec entry for `add-menu-item-to-command-table` says
> Adds a command menu item to command-table's menu. string is the name of the command menu item; **its character case is ignored**.

Reply:
replaced with equalp